c-bindings-gen/src/main.rs

```diff
@@ -995,6 +995,10 @@ fn writeln_enum<'a, 'b, W: std::io::Write>(w: &mut W, e: &'a syn::ItemEnum, type
 	if needs_free {
 		writeln!(w, "#[no_mangle]\npub extern \"C\" fn {}_free(this_ptr: {}) {{ }}", e.ident, e.ident).unwrap();
 	}
+	writeln!(w, "#[no_mangle]").unwrap();
+	writeln!(w, "pub extern \"C\" fn {}_clone(orig: &{}) -> {} {{", e.ident, e.ident, e.ident).unwrap();
```

Comment:
Just to confirm, it's fine that some `pub(enum)`s in RL don't implement `Clone`?

Reply:
Hmm, I mean I guess we should have them match, but its not an issue per se currently - if all the fields support `Clone` (as they would have to for the bindings crate to not fail to compile), then exposing a `Clone` to bindings seems fine.